.github/workflows/test-and-deploy.yml

```diff
@@ -18,13 +18,17 @@ jobs:
           check-latest: true
           cache: npm
       - name: Install prerequisites
-        run: make prereqs
+        run: |
+          make prereqs
+          python3 -m pip install numba
```

Comment:
Choosing on balance to not pin the numba version. Numba tends to support a narrow range of python versions, and it is a rather mature project. I anticipate that the future maintenance burden of bumping a version requirement exceeds the risk of future versions updates failing this little test.

Reply:
Seems reasonable to me.